easybuild/easyconfigs/s/scenicplus/scenicplus-1.0a1-18072024-foss-2023a.eb

```diff
@@ -0,0 +1,281 @@
+easyblock = 'PythonBundle'
+
+name = 'scenicplus'
+version = '1.0a1-18072024'
```

Comment:
datestamp should be `<year><month><day>`, and should correspond to date of commit being used (so in this case `20240513`)